packages/typescript/package.json

```diff
@@ -30,14 +30,14 @@
     "@metamask/eslint-config": "^9.0.0",
     "@typescript-eslint/eslint-plugin": "^5.33.0",
     "@typescript-eslint/parser": "^5.33.0",
-    "eslint": "^7.23.0",
+    "eslint": "^8.21.0",
     "typescript": "^4.0.7"
```

Comment:
We should bump this eventually along with the other typescript packages, but not necessary now.